dev/com.ibm.ws.microprofile.config.1.1/resources/com/ibm/ws/microprofile/config/resources/Config.nlsprops

```diff
@@ -112,6 +112,16 @@ generic.type.variables.notsupported.CWMCG0018E=CWMCG0018E: The {0} Generic Type
 generic.type.variables.notsupported.CWMCG0018E.explanation=An attempt was made to convert using a Generic Type Variable, such as List<T>. This is not supported.
 generic.type.variables.notsupported.CWMCG0018E.useraction=Instead, use a concrete argument type, such as List<String>.
 
+# Failed to start the {0} ConfigSource refresher process.
+failed.to.start.refresher.CWMCG0019E=CWMCG0019E: Failed to start the {0} ConfigSource refresher process.
+failed.to.start.refresher.CWMCG0019E.explanation=The specified ConfigSource will not be properly updated.
+failed.to.start.refresher.CWMCG0019E.useraction=Review the server message.log and FFDC logs to identify the problem.
+
+# Failed to start the {0} ConfigSource refresher process. The exception is: {1}.
+failed.to.start.refresher.exception.CWMCG0020E=CWMCG0020E: Failed to start the {0} ConfigSource refresher process. The exception is: {1}.
+failed.to.start.refresher.exception.CWMCG0020E.explanation=The specified ConfigSource will not be properly updated.
```

Comment:
Change the "The specified ConfigSource will not be properly updated." to "The specified ConfigSource is not be properly updated'.